packages/component/src/ActivityStatus/private/Originator.tsx

```diff
@@ -23,7 +20,7 @@ const Originator = memo(({ replyAction }: Props) => {
       {text}
     </a>
   ) : (
-    <span className={classNames('webchat__activity-status__originator', originatorActivityStatus + '')}>{text}</span>
+    <span className="webchat__activity-status__originator">{text}</span>
```

Comment:
We don't have the `originatorActivityStatus` style set, it is in `activityStatus` style set... caught by types of `useStyleSet`. 😊